### ISSUE TITLE
Incorrect Results for Non-existent Language/Data Type in total Command

### ISSUE THREAD

**Issue by catreedle** [scribe-org/Scribe-Data#307]
### Terms

- [X] I have searched all [open bug reports](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

When a non-existent language or data type is provided to the total command, the system returns an incorrect number of lexemes instead of a "Not found" message.

**Steps to Reproduce**:
1. Run the total command with an invalid language or data type.
2. Observe that the command returns a number instead of indicating that the language or data type doesn't exist.

![bug](https://github.com/user-attachments/assets/88bb9e6e-dc58-4d35-a45b-3cd87cb47cdb)

**Expected Behavior**:
When a non-existent language or data type is provided, the command should return a clear message indicating "Total number of lexemes: Not found" instead of returning incorrect results.

**Proposed Solution**:
- [ ] Add a check function to validate if the provided language and data type exist in the [language_metadata](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/resources/language_metadata.json) or [data_type_metadata](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/resources/data_type_metadata.json).
- [ ] If either the language or data type does not exist, the system should return a "Not found" message without executing the query.

This issue is closely related to #295
This issue is blocked by #295

**Contribution**
I would love to work and collaborate on this.

**Comment by catreedle** [comment]
Hi @andrewtavis, can you please add the blocked label and assign this to me? Thank you. :)

**Comment by DeleMike** [comment]
hey @catreedle! could we collaborate on this issue? we could discuss it over Matrix. what do you think?

**Comment by catreedle** [comment]
Hi @DeleMike. Sure thing, I’d be happy to collaborate on this. Just a heads up that it's currently blocked by another issue, but we can still discuss it. :)

**Comment by DeleMike** [comment]
cool! I'll DM you on Matrix :)

**Comment by andrewtavis** [comment]
Closed via #337, with the follow up work in #341 :) Thanks @catreedle!
